### ISSUE TITLE
Navigation - Consistency

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#4]
Menus or other navigation mechanisms should be located in the same place across the site.

**Why is this required?** People will need to know what to expect. It can vary across devices, but should be consistent within that device.

**Recommendations**: Objects that appear on all web pages should be located in the similar place relative to each other on each page and are identified by the same name
